app/cafe.py

```diff
@@ -1,7 +1,7 @@
+from datetime import date
 from app.errors import (NotVaccinatedError,

```

Comment:
The section of PEP 8 concerning imports recommends grouping them in the following order:

    Standard library imports.
    Related third-party imports.
    Local application/library specific imports.

You should put a blank line between each group of imports.